pkgs/os-specific/linux/cshatag/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoPackage, fetchFromGitHub, attr }:
+
+buildGoPackage rec {
```

Comment:
Would you mind switching to buildGoModule? It does not requires the deps.nix.

Reply:
`buildGoModule` builds a Go module, but this isn't one. This package has only two dependencies, so deps.nix cluttering Nixpkgs isn't much of a concern, here.
Also see https://github.com/NixOS/nixpkgs/issues/84826 for the downsides of `buildGoModule`.

Reply:
I didn't check if it is a Go module or not. Thanks for the explanation.